backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java

```diff
@@ -30,6 +30,7 @@
 @ApplicationScoped
 public class RbacIdentityProvider implements IdentityProvider<RhIdentityAuthenticationRequest> {
 
+    public static final String RBAC_READ_EVENTS_NOTIFICATIONS = "read:events";
```

Comment:
Shouldn't this be named `RBAC_READ_EVENTS` to be consistent with the other roles naming convention?

Reply:
To be really consistent it should be similar to RBAC_READ_INTEGRATIONS_ENDPOINTS. RBAC_READ_NOTICIATIONS_EVENTS i would say.

Reply:
I never had a look at [rbac-config/tree/master/configs](https://github.com/RedHatInsights/rbac-config/tree/master/configs) before today. Now that I did it, I changed my mind and I think `RBAC_READ_NOTIFICATIONS_EVENTS` better reflects the actual RBAC permissions. :smiley: